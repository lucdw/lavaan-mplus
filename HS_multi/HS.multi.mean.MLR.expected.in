

    title:
    		[This syntax is autogenerated]
    		CFA Mplus syntax
    data:
    		file is HS.multi.raw;
                    listwise=on;
    variable:
    		names are
    		school x1  x2  x3  x4  x5  x6
    		x7  x8  x9;
    		missing are all (-999999);
                    grouping is school (1=Pasteur, 2=GrantWhite);
    analysis:
    !               model=nomeanstructure;
                    estimator=MLR;
                    information=expected;
                    convergence=0.00000001;

    model:
                    visual  by x1 x2 x3;
                    textual by x4 x5 x6;
                    speed   by x7 x8 x9;

    model GrantWhite:
            visual by x1@1;
            visual by x2;
            visual by x3;
            textual by x4@1;
            textual by x5;
            textual by x6;
            speed by x7@1;
            speed by x8;
            speed by x9;
            [visual-speed@0];
            [x1-x9];
    Output:  standardized;






