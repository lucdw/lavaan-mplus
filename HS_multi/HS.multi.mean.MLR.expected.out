Mplus VERSION 8.2
MUTHEN & MUTHEN
01/17/2024   7:42 AM

INPUT INSTRUCTIONS



      title:
      		[This syntax is autogenerated]
      		CFA Mplus syntax
      data:
      		file is HS.multi.raw;
                      listwise=on;
      variable:
      		names are
      		school x1  x2  x3  x4  x5  x6
      		x7  x8  x9;
      		missing are all (-999999);
                      grouping is school (1=Pasteur, 2=GrantWhite);
      analysis:
      !               model=nomeanstructure;
                      estimator=MLR;
                      information=expected;
                      convergence=0.00000001;

      model:
                      visual  by x1 x2 x3;
                      textual by x4 x5 x6;
                      speed   by x7 x8 x9;

      model GrantWhite:
              visual by x1@1;
              visual by x2;
              visual by x3;
              textual by x4@1;
              textual by x5;
              textual by x6;
              speed by x7@1;
              speed by x8;
              speed by x9;
              [visual-speed@0];
              [x1-x9];
      Output:  standardized;









INPUT READING TERMINATED NORMALLY




[This syntax is autogenerated]
CFA Mplus syntax

SUMMARY OF ANALYSIS

Number of groups                                                 2
Number of observations
   Group PASTEUR                                               156
   Group GRANTWHITE                                            145
   Total sample size                                           301

Number of dependent variables                                    9
Number of independent variables                                  0
Number of continuous latent variables                            3

Observed dependent variables

  Continuous
   X1          X2          X3          X4          X5          X6
   X7          X8          X9

Continuous latent variables
   VISUAL      TEXTUAL     SPEED

Variables with special functions

  Grouping variable     SCHOOL

Estimator                                                      MLR
Information matrix                                        EXPECTED
Maximum number of iterations                                  1000
Convergence criterion                                    0.100D-07
Maximum number of steepest descent iterations                   20

Input data file(s)
  HS.multi.raw

Input data format  FREE



UNIVARIATE SAMPLE STATISTICS


     UNIVARIATE HIGHER-ORDER MOMENT DESCRIPTIVE STATISTICS FOR PASTEUR

         Variable/         Mean/     Skewness/   Minimum/ % with                Percentiles
        Sample Size      Variance    Kurtosis    Maximum  Min/Max      20%/60%    40%/80%    Median

     X1                    4.941      -0.374       0.667    0.64%       4.000      4.667      5.000
             156.000       1.395       0.674       7.500    1.28%       5.333      5.833
     X2                    5.984       0.688       3.500    0.64%       5.000      5.500      5.750
             156.000       1.504       0.209       9.250    1.92%       6.000      7.000
     X3                    2.487       0.154       0.250    0.64%       1.375      2.000      2.375
             156.000       1.346      -1.078       4.500    3.21%       2.625      3.750
     X4                    2.823       0.219       0.000    0.64%       1.667      2.667      2.667
             156.000       1.320      -0.089       6.000    0.64%       3.000      3.667
     X5                    3.995      -0.130       1.000    0.64%       2.750      3.500      4.000
             156.000       1.708      -0.851       6.750    0.64%       4.500      5.250
     X6                    1.922       1.000       0.143    1.28%       1.000      1.571      1.857
             156.000       0.974       1.932       6.143    0.64%       2.143      2.714
     X7                    4.432       0.300       2.043    0.64%       3.478      4.000      4.370
             156.000       1.170      -0.421       7.435    0.64%       4.696      5.522
     X8                    5.563       0.358       3.500    0.64%       4.750      5.250      5.475
             156.000       0.952       0.050       8.300    0.64%       5.750      6.300
     X9                    5.418       0.223       2.778    0.64%       4.611      5.111      5.417
             156.000       0.978       0.149       8.611    0.64%       5.639      6.222


     UNIVARIATE HIGHER-ORDER MOMENT DESCRIPTIVE STATISTICS FOR GRANTWHITE

         Variable/         Mean/     Skewness/   Minimum/ % with                Percentiles
        Sample Size      Variance    Kurtosis    Maximum  Min/Max      20%/60%    40%/80%    Median

     X1                    4.930      -0.118       1.833    0.69%       4.000      4.667      5.000
             145.000       1.319      -0.085       8.500    0.69%       5.167      6.000
     X2                    6.200       0.237       2.250    0.69%       5.250      5.750      6.250
             145.000       1.226       0.801       9.250    1.38%       6.250      7.000
     X3                    1.996       0.617       0.375    1.38%       1.000      1.625      1.875
             145.000       1.073      -0.480       4.500    0.69%       2.000      3.000
     X4                    3.317       0.401       0.333    0.69%       2.333      3.000      3.000
             145.000       1.257       0.203       6.333    0.69%       3.333      4.333
     X5                    4.712      -0.544       1.000    0.69%       4.000      4.500      4.750
             145.000       1.342       0.172       7.000    0.69%       5.000      5.750
     X6                    2.469       0.721       0.286    0.69%       1.571      2.000      2.286
             145.000       1.280       0.184       5.857    0.69%       2.571      3.286
     X7                    3.921       0.161       1.304    0.69%       2.957      3.565      3.870
             145.000       1.062      -0.385       6.478    0.69%       4.130      4.783
     X8                    5.488       0.691       3.050    0.69%       4.650      5.250      5.500
             145.000       1.094       2.164      10.000    0.69%       5.700      6.250
     X9                    5.327       0.198       3.111    0.69%       4.417      5.056      5.306
             145.000       1.051       0.457       9.250    0.69%       5.583      6.222


THE MODEL ESTIMATION TERMINATED NORMALLY



MODEL FIT INFORMATION

Number of Free Parameters                       60

Loglikelihood

          H0 Value                       -3682.198
          H0 Scaling Correction Factor      1.0628
            for MLR
          H1 Value                       -3624.272
          H1 Scaling Correction Factor      1.0334
            for MLR

Information Criteria

          Akaike (AIC)                    7484.395
          Bayesian (BIC)                  7706.822
          Sample-Size Adjusted BIC        7516.536
            (n* = (n + 2) / 24)

Chi-Square Test of Model Fit

          Value                            116.230*
          Degrees of Freedom                    48
          P-Value                           0.0000
          Scaling Correction Factor         0.9967
            for MLR

Chi-Square Contribution From Each Group

          PASTEUR                           64.519
          GRANTWHITE                        51.711

*   The chi-square value for MLM, MLMV, MLR, ULSMV, WLSM and WLSMV cannot be used
    for chi-square difference testing in the regular way.  MLM, MLR and WLSM
    chi-square difference testing is described on the Mplus website.  MLMV, WLSMV,
    and ULSMV difference testing is done using the DIFFTEST option.

RMSEA (Root Mean Square Error Of Approximation)

          Estimate                           0.097
          90 Percent C.I.                    0.075  0.120
          Probability RMSEA <= .05           0.001

CFI/TLI

          CFI                                0.921
          TLI                                0.881

Chi-Square Test of Model Fit for the Baseline Model

          Value                            934.309
          Degrees of Freedom                    72
          P-Value                           0.0000

SRMR (Standardized Root Mean Square Residual)

          Value                              0.068



MODEL RESULTS

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

Group PASTEUR

 VISUAL   BY
    X1                 1.000      0.000    999.000    999.000
    X2                 0.394      0.131      3.012      0.003
    X3                 0.570      0.148      3.839      0.000

 TEXTUAL  BY
    X4                 1.000      0.000    999.000    999.000
    X5                 1.183      0.108     10.981      0.000
    X6                 0.875      0.088      9.979      0.000

 SPEED    BY
    X7                 1.000      0.000    999.000    999.000
    X8                 1.125      0.286      3.935      0.000
    X9                 0.922      0.204      4.525      0.000

 TEXTUAL  WITH
    VISUAL             0.479      0.119      4.038      0.000

 SPEED    WITH
    VISUAL             0.185      0.069      2.665      0.008
    TEXTUAL            0.182      0.069      2.628      0.009

 Means
    VISUAL             0.000      0.000    999.000    999.000
    TEXTUAL            0.000      0.000    999.000    999.000
    SPEED              0.000      0.000    999.000    999.000

 Intercepts
    X1                 4.941      0.095     52.249      0.000
    X2                 5.984      0.098     60.949      0.000
    X3                 2.487      0.093     26.778      0.000
    X4                 2.823      0.092     30.689      0.000
    X5                 3.995      0.105     38.183      0.000
    X6                 1.922      0.079     24.321      0.000
    X7                 4.432      0.087     51.181      0.000
    X8                 5.563      0.078     71.214      0.000
    X9                 5.418      0.079     68.440      0.000

 Variances
    VISUAL             1.097      0.325      3.374      0.001
    TEXTUAL            0.894      0.158      5.676      0.000
    SPEED              0.350      0.127      2.760      0.006

 Residual Variances
    X1                 0.298      0.263      1.136      0.256
    X2                 1.334      0.165      8.096      0.000
    X3                 0.989      0.121      8.179      0.000
    X4                 0.425      0.073      5.846      0.000
    X5                 0.456      0.090      5.073      0.000
    X6                 0.290      0.056      5.205      0.000
    X7                 0.820      0.126      6.497      0.000
    X8                 0.510      0.100      5.095      0.000
    X9                 0.680      0.108      6.324      0.000

Group GRANTWHITE

 VISUAL   BY
    X1                 1.000      0.000    999.000    999.000
    X2                 0.736      0.162      4.532      0.000
    X3                 0.925      0.179      5.155      0.000

 TEXTUAL  BY
    X4                 1.000      0.000    999.000    999.000
    X5                 0.990      0.088     11.309      0.000
    X6                 0.963      0.091     10.639      0.000

 SPEED    BY
    X7                 1.000      0.000    999.000    999.000
    X8                 1.226      0.162      7.563      0.000
    X9                 1.058      0.142      7.442      0.000

 TEXTUAL  WITH
    VISUAL             0.408      0.105      3.883      0.000

 SPEED    WITH
    VISUAL             0.276      0.082      3.357      0.001
    TEXTUAL            0.222      0.094      2.367      0.018

 Means
    VISUAL             0.000      0.000    999.000    999.000
    TEXTUAL            0.000      0.000    999.000    999.000
    SPEED              0.000      0.000    999.000    999.000

 Intercepts
    X1                 4.930      0.095     51.696      0.000
    X2                 6.200      0.092     67.416      0.000
    X3                 1.996      0.086     23.195      0.000
    X4                 3.317      0.093     35.625      0.000
    X5                 4.712      0.096     48.986      0.000
    X6                 2.469      0.094     26.277      0.000
    X7                 3.921      0.086     45.819      0.000
    X8                 5.488      0.087     63.174      0.000
    X9                 5.327      0.085     62.571      0.000

 Variances
    VISUAL             0.604      0.180      3.352      0.001
    TEXTUAL            0.942      0.162      5.799      0.000
    SPEED              0.461      0.108      4.253      0.000

 Residual Variances
    X1                 0.715      0.167      4.270      0.000
    X2                 0.899      0.140      6.443      0.000
    X3                 0.557      0.107      5.201      0.000
    X4                 0.315      0.066      4.781      0.000
    X5                 0.419      0.071      5.867      0.000
    X6                 0.406      0.076      5.358      0.000
    X7                 0.600      0.080      7.543      0.000
    X8                 0.401      0.112      3.593      0.000
    X9                 0.535      0.084      6.344      0.000


STANDARDIZED MODEL RESULTS


STDYX Standardization

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

Group PASTEUR

 VISUAL   BY
    X1                 0.887      0.108      8.222      0.000
    X2                 0.336      0.084      3.993      0.000
    X3                 0.515      0.082      6.305      0.000

 TEXTUAL  BY
    X4                 0.823      0.038     21.488      0.000
    X5                 0.856      0.032     27.069      0.000
    X6                 0.838      0.030     27.847      0.000

 SPEED    BY
    X7                 0.547      0.090      6.107      0.000
    X8                 0.682      0.081      8.414      0.000
    X9                 0.551      0.075      7.341      0.000

 TEXTUAL  WITH
    VISUAL             0.484      0.101      4.777      0.000

 SPEED    WITH
    VISUAL             0.299      0.096      3.109      0.002
    TEXTUAL            0.325      0.093      3.494      0.000

 Means
    VISUAL             0.000      0.000    999.000    999.000
    TEXTUAL            0.000      0.000    999.000    999.000
    SPEED              0.000      0.000    999.000    999.000

 Intercepts
    X1                 4.183      0.302     13.834      0.000
    X2                 4.880      0.263     18.554      0.000
    X3                 2.144      0.105     20.372      0.000
    X4                 2.457      0.146     16.776      0.000
    X5                 3.057      0.162     18.895      0.000
    X6                 1.947      0.134     14.585      0.000
    X7                 4.098      0.203     20.231      0.000
    X8                 5.702      0.316     18.021      0.000
    X9                 5.480      0.319     17.159      0.000

 Variances
    VISUAL             1.000      0.000    999.000    999.000
    TEXTUAL            1.000      0.000    999.000    999.000
    SPEED              1.000      0.000    999.000    999.000

 Residual Variances
    X1                 0.214      0.191      1.118      0.264
    X2                 0.887      0.057     15.659      0.000
    X3                 0.735      0.084      8.755      0.000
    X4                 0.322      0.063      5.110      0.000
    X5                 0.267      0.054      4.924      0.000
    X6                 0.297      0.050      5.894      0.000
    X7                 0.701      0.098      7.164      0.000
    X8                 0.535      0.110      4.849      0.000
    X9                 0.696      0.083      8.401      0.000

Group GRANTWHITE

 VISUAL   BY
    X1                 0.677      0.088      7.679      0.000
    X2                 0.517      0.071      7.307      0.000
    X3                 0.694      0.072      9.682      0.000

 TEXTUAL  BY
    X4                 0.866      0.034     25.723      0.000
    X5                 0.829      0.034     24.552      0.000
    X6                 0.826      0.037     22.471      0.000

 SPEED    BY
    X7                 0.659      0.057     11.605      0.000
    X8                 0.796      0.051     15.527      0.000
    X9                 0.701      0.055     12.766      0.000

 TEXTUAL  WITH
    VISUAL             0.541      0.095      5.711      0.000

 SPEED    WITH
    VISUAL             0.523      0.101      5.206      0.000
    TEXTUAL            0.336      0.116      2.909      0.004

 Means
    VISUAL             0.000      0.000    999.000    999.000
    TEXTUAL            0.000      0.000    999.000    999.000
    SPEED              0.000      0.000    999.000    999.000

 Intercepts
    X1                 4.293      0.267     16.086      0.000
    X2                 5.599      0.386     14.497      0.000
    X3                 1.926      0.092     20.986      0.000
    X4                 2.959      0.179     16.548      0.000
    X5                 4.068      0.290     14.024      0.000
    X6                 2.182      0.118     18.462      0.000
    X7                 3.805      0.207     18.357      0.000
    X8                 5.246      0.424     12.383      0.000
    X9                 5.196      0.338     15.378      0.000

 Variances
    VISUAL             1.000      0.000    999.000    999.000
    TEXTUAL            1.000      0.000    999.000    999.000
    SPEED              1.000      0.000    999.000    999.000

 Residual Variances
    X1                 0.542      0.119      4.546      0.000
    X2                 0.733      0.073     10.041      0.000
    X3                 0.519      0.099      5.222      0.000
    X4                 0.251      0.058      4.305      0.000
    X5                 0.312      0.056      5.573      0.000
    X6                 0.317      0.061      5.219      0.000
    X7                 0.566      0.075      7.553      0.000
    X8                 0.367      0.082      4.493      0.000
    X9                 0.509      0.077      6.612      0.000


STDY Standardization

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

Group PASTEUR

 VISUAL   BY
    X1                 0.887      0.108      8.222      0.000
    X2                 0.336      0.084      3.993      0.000
    X3                 0.515      0.082      6.305      0.000

 TEXTUAL  BY
    X4                 0.823      0.038     21.488      0.000
    X5                 0.856      0.032     27.069      0.000
    X6                 0.838      0.030     27.847      0.000

 SPEED    BY
    X7                 0.547      0.090      6.107      0.000
    X8                 0.682      0.081      8.414      0.000
    X9                 0.551      0.075      7.341      0.000

 TEXTUAL  WITH
    VISUAL             0.484      0.101      4.777      0.000

 SPEED    WITH
    VISUAL             0.299      0.096      3.109      0.002
    TEXTUAL            0.325      0.093      3.494      0.000

 Means
    VISUAL             0.000      0.000    999.000    999.000
    TEXTUAL            0.000      0.000    999.000    999.000
    SPEED              0.000      0.000    999.000    999.000

 Intercepts
    X1                 4.183      0.302     13.834      0.000
    X2                 4.880      0.263     18.554      0.000
    X3                 2.144      0.105     20.372      0.000
    X4                 2.457      0.146     16.776      0.000
    X5                 3.057      0.162     18.895      0.000
    X6                 1.947      0.134     14.585      0.000
    X7                 4.098      0.203     20.231      0.000
    X8                 5.702      0.316     18.021      0.000
    X9                 5.480      0.319     17.159      0.000

 Variances
    VISUAL             1.000      0.000    999.000    999.000
    TEXTUAL            1.000      0.000    999.000    999.000
    SPEED              1.000      0.000    999.000    999.000

 Residual Variances
    X1                 0.214      0.191      1.118      0.264
    X2                 0.887      0.057     15.659      0.000
    X3                 0.735      0.084      8.755      0.000
    X4                 0.322      0.063      5.110      0.000
    X5                 0.267      0.054      4.924      0.000
    X6                 0.297      0.050      5.894      0.000
    X7                 0.701      0.098      7.164      0.000
    X8                 0.535      0.110      4.849      0.000
    X9                 0.696      0.083      8.401      0.000

Group GRANTWHITE

 VISUAL   BY
    X1                 0.677      0.088      7.679      0.000
    X2                 0.517      0.071      7.307      0.000
    X3                 0.694      0.072      9.682      0.000

 TEXTUAL  BY
    X4                 0.866      0.034     25.723      0.000
    X5                 0.829      0.034     24.552      0.000
    X6                 0.826      0.037     22.471      0.000

 SPEED    BY
    X7                 0.659      0.057     11.605      0.000
    X8                 0.796      0.051     15.527      0.000
    X9                 0.701      0.055     12.766      0.000

 TEXTUAL  WITH
    VISUAL             0.541      0.095      5.711      0.000

 SPEED    WITH
    VISUAL             0.523      0.101      5.206      0.000
    TEXTUAL            0.336      0.116      2.909      0.004

 Means
    VISUAL             0.000      0.000    999.000    999.000
    TEXTUAL            0.000      0.000    999.000    999.000
    SPEED              0.000      0.000    999.000    999.000

 Intercepts
    X1                 4.293      0.267     16.086      0.000
    X2                 5.599      0.386     14.497      0.000
    X3                 1.926      0.092     20.986      0.000
    X4                 2.959      0.179     16.548      0.000
    X5                 4.068      0.290     14.024      0.000
    X6                 2.182      0.118     18.462      0.000
    X7                 3.805      0.207     18.357      0.000
    X8                 5.246      0.424     12.383      0.000
    X9                 5.196      0.338     15.378      0.000

 Variances
    VISUAL             1.000      0.000    999.000    999.000
    TEXTUAL            1.000      0.000    999.000    999.000
    SPEED              1.000      0.000    999.000    999.000

 Residual Variances
    X1                 0.542      0.119      4.546      0.000
    X2                 0.733      0.073     10.041      0.000
    X3                 0.519      0.099      5.222      0.000
    X4                 0.251      0.058      4.305      0.000
    X5                 0.312      0.056      5.573      0.000
    X6                 0.317      0.061      5.219      0.000
    X7                 0.566      0.075      7.553      0.000
    X8                 0.367      0.082      4.493      0.000
    X9                 0.509      0.077      6.612      0.000


STD Standardization

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

Group PASTEUR

 VISUAL   BY
    X1                 1.047      0.155      6.749      0.000
    X2                 0.412      0.111      3.724      0.000
    X3                 0.597      0.102      5.831      0.000

 TEXTUAL  BY
    X4                 0.946      0.083     11.353      0.000
    X5                 1.119      0.072     15.632      0.000
    X6                 0.827      0.080     10.316      0.000

 SPEED    BY
    X7                 0.591      0.107      5.520      0.000
    X8                 0.665      0.099      6.688      0.000
    X9                 0.545      0.084      6.498      0.000

 TEXTUAL  WITH
    VISUAL             0.484      0.101      4.777      0.000

 SPEED    WITH
    VISUAL             0.299      0.096      3.109      0.002
    TEXTUAL            0.325      0.093      3.494      0.000

 Means
    VISUAL             0.000      0.000    999.000    999.000
    TEXTUAL            0.000      0.000    999.000    999.000
    SPEED              0.000      0.000    999.000    999.000

 Intercepts
    X1                 4.941      0.095     52.249      0.000
    X2                 5.984      0.098     60.949      0.000
    X3                 2.487      0.093     26.778      0.000
    X4                 2.823      0.092     30.689      0.000
    X5                 3.995      0.105     38.183      0.000
    X6                 1.922      0.079     24.321      0.000
    X7                 4.432      0.087     51.181      0.000
    X8                 5.563      0.078     71.214      0.000
    X9                 5.418      0.079     68.440      0.000

 Variances
    VISUAL             1.000      0.000    999.000    999.000
    TEXTUAL            1.000      0.000    999.000    999.000
    SPEED              1.000      0.000    999.000    999.000

 Residual Variances
    X1                 0.298      0.263      1.136      0.256
    X2                 1.334      0.165      8.096      0.000
    X3                 0.989      0.121      8.179      0.000
    X4                 0.425      0.073      5.846      0.000
    X5                 0.456      0.090      5.073      0.000
    X6                 0.290      0.056      5.205      0.000
    X7                 0.820      0.126      6.497      0.000
    X8                 0.510      0.100      5.095      0.000
    X9                 0.680      0.108      6.324      0.000

Group GRANTWHITE

 VISUAL   BY
    X1                 0.777      0.116      6.703      0.000
    X2                 0.572      0.094      6.075      0.000
    X3                 0.719      0.091      7.898      0.000

 TEXTUAL  BY
    X4                 0.971      0.084     11.598      0.000
    X5                 0.961      0.083     11.577      0.000
    X6                 0.935      0.082     11.386      0.000

 SPEED    BY
    X7                 0.679      0.080      8.507      0.000
    X8                 0.833      0.089      9.349      0.000
    X9                 0.719      0.086      8.377      0.000

 TEXTUAL  WITH
    VISUAL             0.541      0.095      5.711      0.000

 SPEED    WITH
    VISUAL             0.523      0.101      5.206      0.000
    TEXTUAL            0.336      0.116      2.909      0.004

 Means
    VISUAL             0.000      0.000    999.000    999.000
    TEXTUAL            0.000      0.000    999.000    999.000
    SPEED              0.000      0.000    999.000    999.000

 Intercepts
    X1                 4.930      0.095     51.696      0.000
    X2                 6.200      0.092     67.416      0.000
    X3                 1.996      0.086     23.195      0.000
    X4                 3.317      0.093     35.625      0.000
    X5                 4.712      0.096     48.986      0.000
    X6                 2.469      0.094     26.277      0.000
    X7                 3.921      0.086     45.819      0.000
    X8                 5.488      0.087     63.174      0.000
    X9                 5.327      0.085     62.571      0.000

 Variances
    VISUAL             1.000      0.000    999.000    999.000
    TEXTUAL            1.000      0.000    999.000    999.000
    SPEED              1.000      0.000    999.000    999.000

 Residual Variances
    X1                 0.715      0.167      4.270      0.000
    X2                 0.899      0.140      6.443      0.000
    X3                 0.557      0.107      5.201      0.000
    X4                 0.315      0.066      4.781      0.000
    X5                 0.419      0.071      5.867      0.000
    X6                 0.406      0.076      5.358      0.000
    X7                 0.600      0.080      7.543      0.000
    X8                 0.401      0.112      3.593      0.000
    X9                 0.535      0.084      6.344      0.000


R-SQUARE

Group PASTEUR

    Observed                                        Two-Tailed
    Variable        Estimate       S.E.  Est./S.E.    P-Value

    X1                 0.786      0.191      4.111      0.000
    X2                 0.113      0.057      1.996      0.046
    X3                 0.265      0.084      3.153      0.002
    X4                 0.678      0.063     10.744      0.000
    X5                 0.733      0.054     13.535      0.000
    X6                 0.703      0.050     13.924      0.000
    X7                 0.299      0.098      3.054      0.002
    X8                 0.465      0.110      4.207      0.000
    X9                 0.304      0.083      3.670      0.000

Group GRANTWHITE

    Observed                                        Two-Tailed
    Variable        Estimate       S.E.  Est./S.E.    P-Value

    X1                 0.458      0.119      3.840      0.000
    X2                 0.267      0.073      3.654      0.000
    X3                 0.481      0.099      4.841      0.000
    X4                 0.749      0.058     12.861      0.000
    X5                 0.688      0.056     12.276      0.000
    X6                 0.683      0.061     11.235      0.000
    X7                 0.434      0.075      5.802      0.000
    X8                 0.633      0.082      7.763      0.000
    X9                 0.491      0.077      6.383      0.000


QUALITY OF NUMERICAL RESULTS

     Condition Number for the Information Matrix              0.309E-02
       (ratio of smallest to largest eigenvalue)


     Beginning Time:  07:42:47
        Ending Time:  07:42:47
       Elapsed Time:  00:00:00



MUTHEN & MUTHEN
3463 Stoner Ave.
Los Angeles, CA  90066

Tel: (310) 391-9971
Fax: (310) 391-8971
Web: www.StatModel.com
Support: Support@StatModel.com

Copyright (c) 1998-2018 Muthen & Muthen
